src/locales/en.json

```diff
@@ -18,7 +18,7 @@
     "headerSection": {
       "mainText": "Discover, Explore, Learn",
       "secondaryText": "A resource for kids to find fun and educational materials related to environment issues we are facing today.",
-      "headerButton": "Save the Earth"
+      "headerButton": "knowledge & Games"
```

Comment:
Since kids like game more, you may use "Games & Knowledge". You can take it as a suggestion, which is also for Arabic and Turkish translations.

Reply:
capitalize knowledge

Reply:
ptal